site/src/components/ParameterInput/ParameterInput.stories.tsx

```diff
@@ -42,6 +42,16 @@ Basic.args = {
   }),
 }
 
+export const Boolean = Template.bind({})
+Boolean.args = {
+  schema: createParameterSchema({
+    name: "disable_docker",
+    description: "Disable Docker?",
+    validation_value_type: "bool",
+    default_source_value: "false",
```

Comment:
Do we have testing entities for these objects?

Reply:
No, we don't. Do you think it would be interesting to have?

Reply:
I don't know! I haven't worked extensively with these inputs. If you think it will be used in other stories, great. If not, no worries.